test/net/sourceforge/kolmafia/textui/command/JavaScriptCommandTest.java

```diff
@@ -26,7 +26,7 @@ public void printsNull() {
     public void printsUndefined() {
       String output = execute("undefined");
 
-      assertThat(output, startsWith("Returned: org.mozilla.javascript.Undefined"));
+      assertThat(output, startsWith("Returned: null"));
     }
```

Comment:
As per my previous comment, this test doesn't look like expected behaviour. It looks like it would occlude a function that can meaningfully return either null or undefined, however silly such a function would be.

Reply:
I've proposed a better solution in https://github.com/kolmafia/kolmafia/pull/2291